src/core/EVM/EVMStepExecutor.ts

```diff
@@ -480,7 +484,7 @@ export class EVMStepExecutor extends BaseStepExecutor {
       })
       this.statusManager.updateExecution(step, 'FAILED')
       console.warn(e)
```

Comment:
Let's remove this warn

Reply:
Yes its the only console.warn I think. Makes sense to remove it